ui/raidboss/data/04-sb/dungeon/bardams_mettle.txt

```diff
@@ -0,0 +1,169 @@
+2071.4 "--adds spawn--" sync /03:........:Added new combatant Corpsecleaner Eagle/ window 71.4,5
+2078.0 "Pinion" sync /:Yol Feather:1F11:/
+2088.0 "Pinion" sync /:Yol Feather:1F11:/
+2098.0 "Pinion" sync /:Yol Feather:1F11:/
+2108.0 "Pinion" sync /:Yol Feather:1F11:/
+2118.0 "Pinion" sync /:Yol Feather:1F11:/ jump 2078.0
+2128.0 "Pinion"
+2138.0 "Pinion"
+2148.0 "Pinion"
+
+# Unfortunately, because there are two Corpsecleaner Eagles,
+# we can't just sync to a 04 log line.
+2189.7 "--targetable--" sync /22:........:Yol:........:Yol:01/ window 120,5
+2196.8 "Feathercut" sync /:Yol:1F09:/
+2204.0 "Wind Unbound" sync /:Yol:1F0A:/
+2212.7 "Pinion" sync /:Yol Feather:1F11:/
+2214.5 "Flutterfall" sync /:Yol:1F0C:/
+2215.8 "Feathercut" sync /:Yol:1F09:/
+2222.6 "Pinion" sync /:Yol Feather:1F11:/
+2223.3 "Flutterfall" sync /:Yol:1F0C:/
+2230.7 "Wind Unbound" sync /:Yol:1F0A:/
+2234.9 "Feathercut" sync /:Yol:1F09:/
+2239.4 "Pinion" sync /:Yol Feather:1F11:/
+2242.5 "Eye Of The Fierce" sync /:Yol:1F0D:/
+2247.7 "Feathercut" sync /:Yol:1F09:/
+2249.4 "Pinion" sync /:Yol Feather:1F11:/
+
+2253.8 "--untargetable--" sync /22:........:Yol:........:Yol:00/ window 60,5
+2260.3 "Flutterfall" sync /:Yol:1F10:/
+2267.5 "Wingbeat" sync /:Yol:1F0F:/
+2276.9 "Flutterfall" sync /:Yol:1F0C:/
+2284.3 "Wingbeat" sync /:Yol:1F0F:/
+2284.7 "Pinion" sync /:Yol Feather:2593:/
+2293.8 "Flutterfall" sync /:Yol:1F0C:/
+2301.2 "Wingbeat" sync /:Yol:1F0F:/
+2301.6 "Pinion" sync /:Yol Feather:2593:/
+2310.7 "Flutterfall" sync /:Yol:1F0C:/
+2318.2 "Wingbeat" sync /:Yol:1F0F:/
+2318.6 "Pinion" sync /:Yol Feather:2593:/ jump 2284.7
+2327.7 "Flutterfall"
+2335.1 "Wingbeat"
+2335.5 "Pinion"
+2344.6 "Flutterfall"
+2352.1 "Wingbeat"
+2352.5 "Pinion"
+
+
+2361.7 "--targetable--" sync /22:........:Yol:........:Yol:01/ window 110,5
+2368.8 "Feathercut" sync /:Yol:1F09:/
+2376.0 "Wind Unbound" sync /:Yol:1F0A:/
+2384.7 "Pinion" sync /:Yol Feather:1F11:/
+2386.6 "Flutterfall" sync /:Yol:1F0C:/
+2388.0 "Feathercut" sync /:Yol:1F09:/
+2394.7 "Pinion" sync /:Yol Feather:1F11:/
+2395.5 "Flutterfall" sync /:Yol:1F0C:/
+2402.9 "Wind Unbound" sync /:Yol:1F0A:/
+2407.1 "Feathercut" sync /:Yol:1F09:/
+2411.6 "Pinion" sync /:Yol Feather:1F11:/
+2414.7 "Eye Of The Fierce" sync /:Yol:1F0D:/
+2419.9 "Feathercut" sync /:Yol:1F09:/
+2421.5 "Pinion" sync /:Yol Feather:1F11:/
+2433.1 "Feathercut" sync /:Yol:1F09:/
+2440.2 "Wind Unbound" sync /:Yol:1F0A:/
+2448.9 "Pinion" sync /:Yol Feather:1F11:/
+2450.7 "Flutterfall" sync /:Yol:1F0C:/
+2452.1 "Feathercut" sync /:Yol:1F09:/
+2458.9 "Pinion" sync /:Yol Feather:1F11:/
+2459.6 "Flutterfall" sync /:Yol:1F0C:/
+2467.0 "Wind Unbound" sync /:Yol:1F0A:/
+2471.1 "Feathercut" sync /:Yol:1F09:/
+2475.7 "Pinion" sync /:Yol Feather:1F11:/
+2478.7 "Eye Of The Fierce" sync /:Yol:1F0D:/
+2483.9 "Feathercut" sync /:Yol:1F09:/
+2485.7 "Pinion" sync /:Yol Feather:1F11:/
+2490.0 "--untargetable--" sync /22:........:Yol:........:Yol:00/ jump 2253.8 window 130,5
```

Comment:
WAIT this can happen again???

Reply:
Indeed it can. I've never seen it before but it does rotate those phases until 0%.